tools/travis/run_codespell.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+sudo pip install codespell
+codespell -S "*.xz,*.zip,*.bz2,*.7z,*.gz,*.deb,*.rpm,*.snap,*.gpg,*.pyc,*.png,*.ico,*.jar,./.git" -w || true
```

Comment:
the `|| true` at the end of this will hide errors, I suggest we remove it.

Reply:
okay

Reply:
however the return value is the number of typos found, which will be non-zero and cause the build to fail

Reply:
Making it 0 should be ideal :smile:
If we don't make it fail, then we will never detect any new issues.

Reply:
As seen here [https://travis-ci.org/snapcore/snapcraft/jobs/308985647](https://travis-ci.org/snapcore/snapcraft/jobs/308985647), the build fails because codespell returns the number of typos found.